test/system/src/test/java/io/pravega/test/system/ConsumptionBasedRetentionWithMultipleReaderGroupsTest.java

```diff
@@ -283,4 +286,140 @@ public void multipleSubscriberCBRTest() throws Exception {
                 5000, 3 * 60 * 1000L);
 
     }
+
+
+    @Test
+    public void updateRetentionPolicyForCBRTest() throws Exception {
+        final ClientConfig clientConfig = Utils.buildClientConfig(controllerURI);
+        assertTrue("Creating scope", streamManager.createScope(SCOPE_1));
+        assertTrue("Creating stream", streamManager.createStream(SCOPE_1, STREAM_1,
+                StreamConfiguration.builder()
+                        .scalingPolicy(ScalingPolicy.fixed(1))
+                        .retentionPolicy(RetentionPolicy.bySizeBytes(MIN_SIZE_IN_STREAM, MAX_SIZE_IN_STREAM)).build()));
```

Comment:
Also add the case of Time based and share the details of system test results.

Reply:
Added the time based also.